blueocean-web/package.json

```diff
@@ -30,31 +28,41 @@
     "giti": "1.1.3",
     "gulp": "3.9.1",
     "jquery-detached": "2.1.4-v4",
-    "moment": "2.15.1",
     "ncp": "2.0.0",
-    "react-router": "2.8.1",
     "reactify": "1.1.1",
     "zombie": "4.2.1"
   },
   "dependencies": {
+    "@jenkins-cd/blueocean-core-js": "0.0.170",
+    "@jenkins-cd/design-language": "0.0.158",
+    "@jenkins-cd/js-extensions": "0.0.42",
     "@jenkins-cd/js-modules": "0.0.10",
     "history": "2.0.2",
     "immutable": "3.8.1",
     "keymirror": "0.1.1",
     "mobx": "2.6.0",
     "mobx-react": "3.5.7",
     "mobx-react-devtools": "4.2.6",
+    "moment": "2.15.1",
     "react": "15.4.2",
     "react-addons-css-transition-group": "15.4.2",
     "react-dom": "15.4.2",
     "react-i18next": "1.10.1",
     "react-redux": "4.4.5",
+    "react-router": "2.8.1",
     "redux": "3.6.0",
     "redux-thunk": "2.1.0"
   },
   "jenkinscd": {
     "import": [
-      "moment@any"
+      "@jenkins-cd/blueocean-core-js@any",
+      "@jenkins-cd/design-language@any",
+      "@jenkins-cd/js-extensions@any",
+      "moment@any",
+      "react@any",
+      "react-addons-css-transition-group@any",
+      "react-dom@any",
+      "react-router@any"
```

Comment:
These used to be import, but I think they were removed in HPI PR. Can you explain why they were added back?

Reply:
yes, for backwards compatibility, I had to remove auto-import of things that weren't being exported in BO < 1.4. also in order to make it very clear for anyone looking, I added all the imports to the `package.json` files, though there ends up being some duplication that doesn't really hurt anything but displays a warning at times